### PR TITLE
Cloud form cleanup

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* moved cloud forms to use new opt-in fields
* got cloud forms to use a single new form
* got client forms to use a single new form
* redirected all old contact-us forms to new ones
* added a get parameter for thank-you pages to mimic the contact-us logic
* remove dead tender forms

## QA

1. go to all /cloud contact forms and see that it goes to the new forms
2. go to /desktop/contact-us and see that still works
3. go to /tablet/contact-us and see that it still works
4. go to /internet-of-things contact us form and see that it still works
5. go to /server/contact-us and see that it still works

## Issue / Card

[trello card](https://trello.com/c/lOUBdXoy)

### CODE REVIEW

**Comment by pmahnke** [comment]
[Newsletter opt-in form changes across website](https://trello.com/c/lOUBdXoy/2164-newsletter-opt-in-form-changes-across-website)

File: templates/tablet/contact-us.html [hunk 1]
```diff
@@ -9,95 +9,25 @@
...
 
 {% block content %}
-<section class="row row-hero no-border strip-light">
-  <div class="strip-inner-wrapper">
-    <div class="eight-col">
-      <h1>Contact us</h1>
-      <p class="eight-col">If you are thinking about partnering with Ubuntu to power your mobile devices, please fill in your details below and a member of our team will get in touch.</p>
+<section class="row row-hero no-border no-margin-bottom no-padding-bottom strip-light">
+    <div class="strip-inner-wrapper">
+        {% include "shared/_client-contact-us-form.html" with     h1="Contact us"    intro_text="If you are thinking about partnering with Ubuntu to power your mobile devices, please fill in your details below and a member of our team will get in touch."    formid="1371"   lpId="2436"    returnURL="https://www.ubuntu.com/tablet/thank-you"    %}
```
Comment (nottrobin) [inline templates/tablet/contact-us.html]: Why so much space before `h1=`?`

**Comment by nottrobin** [comment]
Should "Contact us about your custom engineering needs ›" on <http://127.0.0.1:8001/cloud/openstack> not have a `product=` of some kind?

File: templates/internet-of-things/contact-us.html [hunk 2]
```diff
@@ -8,23 +8,41 @@
...
+            {% include "shared/_client-contact-us-form.html" with     h1="Contact us about digital signage"    intro_text="If you are thinking about building a digital signage product on Ubuntu, please fill in your details below and a member of our team will get in touch."    formid="1422"   lpId="2166"    returnURL="http://ubuntu.com/internet-of-things/thank-you"    %}
 
-    {% elif product == 'gateways' %}
+        {% elif product == 'gateways' %}
 
-        {% include "internet-of-things/shared/_contact_us_form.html" with     h1="Contact us about gateways"    intro_text="If you would like to know more about Ubuntu for edge gateways please fill in your details below and a member of our team will get in touch."    formid="1663"   lpId="3143"    returnURL="http://ubuntu.com/internet-of-things/thank-you"    %}
+            {% include "shared/_client-contact-us-form.html" with     h1="Contact us about gateways"    intro_text="If you would like to know more about Ubuntu for edge gateways please fill in your details below and a member of our team will get in touch."    formid="1663"   lpId="3143"    returnURL="http://ubuntu.com/internet-of-things/thank-you"    %}
```
Comment (nottrobin) [inline templates/internet-of-things/contact-us.html]: Again, why so much space before the arguments?

**Pull request merged**